playbooks/webapp/README.md

```diff
@@ -137,25 +137,25 @@ To delete the webapp:
 * **rds_instance_name** (str): Name for the database. Default: `mysampledb123`
 * **rds_engine** (str): Engine to use for the database. Default: `postgres`
 * **rds_engine_version** (str): Version number of the database engine to use. Default: `"14.8"`
```

Comment:
Why don't these start with the role name prefix (I noticed there are other variables keeping the old name)?